generator/templates/scss/src/assets/scss/vendors/bootstrap-vue/index.scss

```diff
@@ -0,0 +1,104 @@
+//compose your bootstrap-vue css build
```

Comment:
Does this need to be included? We can probably use the imports from the updateApp function to replace this activity